hack/verify-lualint.sh

```diff
@@ -19,3 +19,5 @@ set -o nounset
 set -o pipefail
 
 luacheck --codes -q rootfs/etc/nginx/lua/
+
+find rootfs/etc/nginx/lua/ -name "*.lua" -not -path "*/test/*" -exec lj-releng -L -s {} + && echo "lj-releng validation is success!"
```

Comment:
Does this also enforce a constraint on the line length?

Reply:
@ElvinEfendi  How about using the `luacheck` tool for restrictions? Limited to 100 characters.

Reply:
thanks 👍